cmd/minishift/cmd/root.go

```diff
@@ -45,22 +49,9 @@ import (
 	"github.com/spf13/viper"
 )
 
-var dirs = [...]string{
-	constants.Minipath,
-	constants.MakeMiniPath("certs"),
-	constants.MakeMiniPath("machines"),
-	constants.MakeMiniPath("cache"),
-	constants.MakeMiniPath("cache", "iso"),
-	constants.MakeMiniPath("cache", "oc"),
-	constants.MakeMiniPath("cache", "images"),
-	constants.MakeMiniPath("config"),
-	constants.MakeMiniPath("addons"),
-	constants.MakeMiniPath("logs"),
-	constants.MakeMiniPath("tmp"),
-}
-
 const (
 	showLibmachineLogs    = "show-libmachine-logs"
+	profile               = "profile"
```

Comment:
Why do we need this?

Reply:
It is required for the persistent flag `RootCmd.PersistentFlags().String(profile`. Check the init()

Reply:
@LalatenduMohanty 
can you explain to me why this is needed?